.github/workflows/docker-build-test.yml

```diff
@@ -18,17 +19,24 @@ jobs:
         uses: actions/checkout@v2 # If you're using actions/checkout@v2 you must set persist-credentials to false in most cases for the deployment to work correctly.
         with:
           persist-credentials: false
+
       - name: Setup Node.js
         uses: actions/setup-node@v2.1.5
         with:
-          node-version: 14.x
-      - name: Install pnpm
-        run: npm i -g pnpm@latest-7
+            node-version: '20.x'
+
+      - name: Enable Corepack
+        run: corepack enable
+
+      - name: Prepare pnpm using Corepack
+        run: corepack prepare pnpm@latest --activate
```

Comment:
IMHO that's a bit dangerous, as this makes builds no longer 100% repeatable
I would recommend setting a pnpm version in the top level package.json, as described [here](https://nodejs.org/api/packages.html#packagemanager)
Also see [here](https://github.com/MiSArch/frontend/blob/main/package.json#L58) for an example

Reply:
I've set the version in previous commits already. I've also thought that latest --activate will download the latest available version and thus changed something (newest commits), but then I've noticed that all actions use pnpm@9.6.0 and decided to check. There is no difference between using prepare pnpm@latest --activate or prepare pnpm as both will download the version specified in the package.json. What would you prefer?

Reply:
I guess the latter one, but your decision